edge-apps/welcome-app/src/assets/main.scss

```diff
@@ -0,0 +1,59 @@
+/* stylelint-disable selector-class-pattern, color-hex-length, at-rule-no-unknown, media-feature-range-notation, at-rule-empty-line-before */
+
+@use 'blueprint/scss/base.scss' as *;
+
+/* Font Details - Inter - 600 (SemiBold) , 500(Medium) and 400(Regular) */
+@font-face {
+  font-family: Inter;
+  src: url('./font/Inter-Regular.woff2') format('woff2');
+  font-weight: 400;
+  font-style: normal;
+  font-display: swap;
+}
+
+@font-face {
+  font-family: Inter;
+  src: url('./font/Inter-Medium.woff2') format('woff2');
+  font-weight: 500;
+  font-style: normal;
+  font-display: swap;
+}
+
+@font-face {
+  font-family: Inter;
+  src: url('./font/Inter-SemiBold.woff2') format('woff2');
+  font-weight: 600;
+  font-style: normal;
+  font-display: swap;
+}
+
+body {
+  font-family: Inter, sans-serif;
+}
+
+/* Colors (Theme) */
+:root {
+  --theme-color-primary: #7E2CD2;
+  --theme-color-secondary: #454BD2;
+  --theme-color-tertiary: #FFF;
+  --theme-color-background: #C9CDD0;
+  --theme-color-card: #F2F5F7;
+
+  /* Padding, Gap, Border-Radius */
+  --custom-4k-padding: 5.714rem;
+  --custom-4k-gap: 5.714rem;
+  --custom-4k-border-radius-big: 6.857rem;
+  --custom-4k-border-radius-medium: 5.429rem;
+  --hd-padding: 2.857rem;
+  --hd-gap: 2.857rem;
+  --hd-border-radius-big: 3.429rem;
+  --hd-border-radius-medium: 2.714rem;
+  --custom-720-padding: 2.286rem;
+  --custom-720-gap: 2.286rem;
+  --custom-720-border-radius-big: 2.429rem;
+  --custom-720-border-radius-medium: 1.714rem;
+  --pi-padding: 1rem;
+  --pi-gap: 1rem;
+  --pi-border-radius-big: 1.571rem;
+  --pi-border-radius-medium: 1.143rem;
+}
```

Comment:
In my open PR for the simple message app migration, I declared them as SCSS variables. If possible, make use of SCSS variables.

You could create a new file called `variables.scss` inside `src/assets` and declare variables there.